docs/hopsworksai/aws/upgrade.md

```diff
@@ -0,0 +1,186 @@
+      <img src="../../../assets/images/hopsworksai/aws/aws-instance-profile.png" alt="Getting the name of your instance profile">
+    </a>
+    <figcaption>Getting the name of your instance profile</figcaption>
+  </figure>
+</p>
+
+
+Once you get your instance profile name, navigate to [AWS management console](https://console.aws.amazon.com/iam/home#), then click on *Roles* and then search for your role name and click on it.  Go to the *Permissions* tab, click on *Add inline policy*, and then go to the *JSON* tab. Paste the following snippet, click on *Review policy*, name it, and click *Create policy*.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "UpgradePermissions",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DetachVolume",
+                "ec2:AttachVolume",
+                "ec2:ModifyInstanceAttribute"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+## Step 3: Run the upgrade process
+
+You need to click on *Upgrade* to start the upgrade process. You will be prompted with the screen shown below to confirm your intention to upgrade: 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/aws/aws-upgrade-prompt-1.png">
+      <img src="../../../assets/images/hopsworksai/aws/aws-upgrade-prompt-1.png" alt="AWS Upgrade Prompt">
+    </a>
+    <figcaption>Upgrade confirmation</figcaption>
+  </figure>
+</p>
+
+Check the *Yes, upgrade cluster* checkbox to proceed, then the *Upgrade* button will be activated as shown below:
+
+!!! warning
+    Currently, we only support upgrade for the head node and you will need to recreate your workers once the upgrade is successfully completed. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/aws/aws-upgrade-prompt-2.png">
+      <img src="../../../assets/images/hopsworksai/aws/aws-upgrade-prompt-2.png" alt="AWS Upgrade Prompt">
+    </a>
+    <figcaption>Upgrade confirmation</figcaption>
+  </figure>
+</p>
+
+Depending on how big your current cluster, the upgrade process may take from 1 hour to a few hours until completion.
+
+!!! note
+    We don't delete your old cluster until the upgrade process is successfully completed. 
+
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/aws/aws-upgrade-start.png">
+      <img src="../../../assets/images/hopsworksai/aws/aws-upgrade-start.png" alt="AWS Upgrade starting">
+    </a>
+    <figcaption>Upgrade is running</figcaption>
+  </figure>
+</p>
+
+Once the upgrade is completed, you can confirm that you have the new Hopsworks version by checking the *Details* tab of your cluster as below:
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/aws/aws-upgrade-complete.png">
+      <img src="../../../assets/images/hopsworksai/aws/aws-upgrade-complete.png" alt="AWS Upgrade complete">
```

Comment:
Can you also add a box surrounding the version as you did in Azure documentation?